### ISSUE TITLE
i have gx6605s set top box can i boot .img using this satelite box or not?

### ISSUE THREAD

**Issue by maxmod** [c-sky/forum#67]
Please answer i have gx6605s set top box can i boot .img using this **satelite box** or not?
do i need to flash this https://github.com/c-sky/tools/blob/master/gx6605s/loader-sflash.bin
please please answer its a china box 4mb 64 mb ram 
thank you

**Comment by maxmod** [comment]
please reply

**Comment by guoren83** [comment]
loader-sflash.bin could boot vmlinux, you can try it. But please reserve your binary first.

**Comment by maxmod** [comment]
Sir,
Thank you so much for reply vmlinux means this file https://gitlab.com/c-sky/buildroot/-/jobs/24633630/artifacts/raw/output/images/usb.img m i right

you mean 1st i flash loader-sflash.bin via system upgrade in **set top box [satelite dish tv box]** than flash usb.img to usb drive and insert into set top box ?
if i am wrong please guide
Thank you so much

**Comment by maxmod** [comment]
getting error on flashing loader-sflash.bin please check screenshot
Thank You
![screen](https://user-images.githubusercontent.com/25316260/50844259-627ba700-1390-11e9-99ef-170a1c1e0b20.jpeg)

**Comment by guoren83** [comment]
You need use uart to flush the bin. See :
https://github.com/c-sky/tools/tree/master/gx6605s

You could google translate from chinese to english :-P

**Comment by guoren83** [comment]
If you broke your top set, don't blame me.

**Comment by maxmod** [comment]
ok thanks

**Comment by maxmod** [comment]
no issue

**Comment by maxmod** [comment]
when i burn usb.img to pen drive and inserting in set top box it do nothing do i need to use 
Gx6605s-generic-sflash.boot Burned auxiliary firmware program may be firmware not allowed to boot or i need to flash .bin via uart ?

**Comment by maxmod** [comment]
what you think problem with binary or bootloader Gx6605s-generic-sflash.boot  bootloader
 or Loader-sflash.bin

**Comment by maxmod** [comment]
As i know bootloader load software so it is possible to flash **Gx6605s-generic-sflash.boot** so when i insert pen drive it load from pen drive and when i remove pen drive it simply load my dth tv

**Comment by maxmod** [comment]
i am beginner and learning

**Comment by maxmod** [comment]
> 
> 
> If you broke your top set, don't blame me.

i don't blame you just help us

**Comment by maxmod** [comment]
do i need to flash both files **Gx6605s-generic-sflash.boot**  and **loader-sflash.bin** via uart

**Comment by maxmod** [comment]
one last question after booting from usb.img can i access dvs 2 using dvb software in linux please reply

**Comment by guoren83** [comment]
Not ready now ...

**Comment by maxmod** [comment]
> 
> 
> Not ready now ...

My gx6605s card set top box has dvb s2 inbuilt so i can't view tv channel using dvbviewer or any other software?
my goal is to make a epg downloader [Electronic Program Guide] it download so i can't build using this ?

**Comment by guoren83** [comment]
That's not csky linux issue. You can ask nationalchip.com

**Comment by maxmod** [comment]
Sir,
here github.com/c-sky/buildroot/blob/master/CHANGES it mention 
after installing dvb-apps can i access or not?
can you suggest box by which i can download epg tv guide directly from satelite i am building a software 
Thank You

**Comment by guoren83** [comment]
I'm not dvb devloper, I'm a CPU developer.

gx6605s is a SOC which use C-SKY CPU 610 and we are working on CPU linux arch porting for this. Your question is out of my capbitlty.

**Comment by maxmod** [comment]
> 
> 
> I'm not dvb devloper, I'm a CPU developer.
> 
> gx6605s is a SOC which use C-SKY CPU 610 and we are working on CPU linux arch porting for this. Your question is out of my capbitlty.

Thank you

**Comment by zerogaber** [comment]
Working like a charm on KOQIT K1 MINI (GX6605S) dvb-s2. For other models pls change keymap.xml manualy. I think this thread can be closed.

**Comment by budiarno** [comment]
> Working like a charm on KOQIT K1 MINI (GX6605S) dvb-s2. For other models pls change keymap.xml manualy. I think this thread can be closed.

![Screenshot from 2019-12-28 05-31-39](https://user-images.githubusercontent.com/15664372/71535265-39ed1400-2937-11ea-9689-fb6e2fe4387f.png)
Do you have vmlinux to load from USB?

**Comment by linuxstv** [comment]
Confirmed:
Working fine on my Getmecom Guoxin GX6605S STB.

**Comment by aklitt** [comment]
Hello,
Working dvb-s2 with RCU in this Linux Firmware ?
Can i Oscam adding in this Linux Firmware ?
Is this Mipsel ?

**Comment by bansal55** [comment]
> Confirmed: Working fine on my Getmecom Guoxin GX6605S STB.

Can you share link 4mb spi flash software for booting vmLinux by usb ,please

**Comment by HimanshuAttri** [comment]
yes but it may be an irreversible step and normal operation of setup box may not work unless u flash the original  firmware using  Rx and Tx   port, in simple words it will just boot linux from usb after that

**Comment by HimanshuAttri** [comment]
@bansal55  yes but it may be an irreversible step and normal operation of setup box may not work unless u flash the original firmware using Rx and Tx port, in simple words it will just boot linux from usb after that

**Comment by bansal55** [comment]
> @bansal55 yes but it may be an irreversible step and normal operation of setup box may not work unless u flash the original firmware using Rx and Tx port, in simple words it will just boot linux from usb after that

Yes I know it won't work as normal settopbox Thanks for replying i have programmer as well as rs232 for flashing using loader and also how to do it i need only 4mb spi flash software for booting Linux by usb , please share me link of software

**Comment by HimanshuAttri** [comment]
@bansal55 here you go 
https://cop-image-prod.oss-cn-hangzhou.aliyuncs.com/resource/420262990181302272/1584685214831/usb.img

**Comment by bansal55** [comment]
> @bansal55 here you go https://cop-image-prod.oss-cn-hangzhou.aliyuncs.com/resource/420262990181302272/1584685214831/usb.img

Thank you ,this one for installing in usb but I also need sflash -loader.bin 4mb link

**Comment by HimanshuAttri** [comment]
@bansal55  https://github.com/c-sky/tools/blob/master/gx6605s/loader-sflash.bin
here it is

**Comment by bansal55** [comment]
> @bansal55 https://github.com/c-sky/tools/blob/master/gx6605s/loader-sflash.bin here it is

This link not working please check ,if you have backup please send me link

**Comment by HimanshuAttri** [comment]
@bansal55 try this i have uploaded https://github.com/HimanshuAttri/ScriptDump/raw/master/loader-sflash.bin

**Comment by bansal55** [comment]
> @bansal55 try this i have uploaded https://github.com/HimanshuAttri/ScriptDump/raw/master/loader-sflash.bin

Thanks mate this one not working for me after flashing it board not booting,the file is just one 1kb

**Comment by linuxstv** [comment]
> > Confirmed: Working fine on my Getmecom Guoxin GX6605S STB.
> 
> Can you share link 4mb spi flash software for booting vmLinux by usb ,please

I used the tutorial here, and it worked just fine:
https://github.com/c-sky/tools/tree/master/gx6605s

**Comment by ManBD1975** [comment]
I am looking for some firmware developer for FTA receiver. Interested peoples are requested to contact me.

**Comment by ManBD1975** [comment]
anyone here to develop GX6605S-NK firmware. I will pay for it. please reply.
